src/main.rs

```diff
@@ -1,87 +1,75 @@
-#![allow(dead_code, unused_variables, unused_imports)]
-pub mod accelerator;
+pub mod driver;
```

Comment:
Now we have `driver` and `drivers` those are too close in spelling and will cause issues (mostly in the vein of pains in the behind when trying to auto-complete).